operations/configuration_management/iSCSI_SBPS_Node_Personalization.md

```diff
@@ -0,0 +1,359 @@
+# iSCSI SBPS worker node(s) personalization
+
+As part of the iSCSI SBPS solution, we need to setup/ configure worker nodes as iSCSI targets (servers) with necessary provisioning, configuration and enablement of required components.
+In order to achieve this, CFS based Ansible plays (csm-config) are created and deployed in order to
+* provision `iSCSI targets`/ `LIO services` and present the `LIO network` service via the HSN and NMN IP networks (TCP port 3260 by default), via iSCSI Portals.
+* enable SBPS Marshal Agent by installing the agent rpm and starting the respective systemd service (`sbps-marshal`)
+* enable `spire` for authentication used for `IMS/ S3` access
+* create `DNS SRV and A records` in order to discover iSCSI targets from iSCSI initiators/clients (passed as boot parameter)
+* mount s3 images (`boot-images` bucket) with new, dedicated s3 user (`ISCSI-SBPS`) read only bucket policy via s3fs
+* apply k8s label (`iscsi=sbps`), to be consumed by the Goss test suit and LIO Metrics for identification of nodes on which these have to run.
+
+Target worker node selection is via dynamic inventory stored in HSM (Hardware State Manager) – the default is all worker nodes (`Management_Worker` group).
+CFS Ansible plays here refers to `Management_Worker` group in the HSM.
+
+The LUN projection is over either HSN or NMN.
+
+## Prerequisites
+
+* Respective csm-config version CFS Ansible plays are available at VCS (Version Control Service)
+* On cluster nodes, the VCS can be accessed through the gateway. VCS credentials for the `crayvcs` user are required before cloning a repository.
+  See [VCS Administrative User](Version_Control_Service_VCS.md#vcs-administrative-user) in
+  [Version Control Service (VCS)](Version_Control_Service_VCS.md) for more information.
+
+Below is the procedure (with examples) that need to be followed for worker node personalization for iSCSI SBPS. Please note that the "Optional" steps mentioned below
+are only to get the latest version of the `csm-config` or to check if the known/ intended version of the `csm-config` is available before proceeding with node personalization
+with creating the CFS config/ CFS session.
+
+## Procedure
+
+### **Check the VCS availability**
+
+```
+ncn-m001:/home # /usr/local/bin/cmsdev test -q vcs
+Starting main run, version: 1.22.0, tag: hMUDa
+Starting sub-run, tag: hMUDa-vcs
+Ended sub-run, tag: hMUDa-vcs (duration: 3.982640351s)
+Ended run, tag: hMUDa (duration: 3.994254571s)
+SUCCESS: All 1 service tests passed: vcs
+ncn-m001:/home #

```

Comment:
```suggestion

```
This is just my opinion and it works as is, but I would remove the prompt at the end of each block. I do not think you need to leave the `ncn-m001:/home #` at the end of the output. This is the same for the rest of the blocks of output.

Reply:
Fixed